staging/src/k8s.io/apiserver/pkg/storage/etcd/etcd_helper.go

```diff
@@ -123,9 +122,6 @@ func (h *etcdHelper) Create(ctx context.Context, key string, obj, out runtime.Ob
 	if err != nil {
 		return err
 	}
-	if version, err := h.versioner.ObjectResourceVersion(obj); err == nil && version != 0 {
-		return errors.New("resourceVersion may not be set on objects to be created")
```

Comment:
Why would we start allowing this? We want to help users not submit data that they sourced incorrectly, and if RV is set, it's probably been sourced incorrectly.

Reply:
I think resourceversion like e.g. creationTimestamp, they should not influence creation. During creation, user should make sure the source correct.